apis/core/v1alpha1/groupversion_info.go

```diff
@@ -0,0 +1,21 @@
+// The package v1alpha1 contains API Schema definitions for the v1alpha1 version of the core Page Operator API group,
```

Comment:
This should be "Package Operator API group" right?

Reply:
Yes! :man_facepalming: